src/lib/testing/cypress/README.md

```diff
@@ -64,3 +64,302 @@ import { waitForAngular } from '@td-vantage/ui-platform/testing/cypress';
 
 waitForAngular();
 ```
+
```

Comment:
We should have 2 sub directories one for utilities and one for mock
@td-vantage/ui-platform/testing/cypress/mock
@td-vantage/ui-platform/testing/cypress/utilities

Each dir can have it's own dir to avoid having such as a big README here

Reply:
Done